Client/Ecosia/UI/EmptyBookmarksView.swift

```diff
@@ -0,0 +1,179 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+final class EmptyBookmarksView: UIView, NotificationThemeable {
+
+    required init?(coder: NSCoder) {
+        assertionFailure("This view is only supposed to be instantiated programmatically")
+        return nil
+    }
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = .localized(.noBookmarksYet)
+        label.textAlignment = .center
+        label.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: .systemFont(ofSize: 17, weight: .semibold))
+        return label
+    }()
+    
+    private let containerStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        return stackView
+    }()
+    
+    private let learnMoreButton: UIButton = {
+       let button = UIButton()
+        button.setTitle(.localized(.learnMore), for: .normal)
+        button.setContentCompressionResistancePriority(.required, for: .horizontal)
+        return button
+    }()
+    
+    private let importBookmarksButton: UIButton = {
+       let button = UIButton()
+        button.setTitle(.localized(.importBookmarks), for: .normal)
+        button.layer.borderWidth = 1
+        button.layer.cornerRadius = 20
+        button.setInsets(
+            forContentPadding: UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10),
+            imageTitlePadding: 0
+        )
+        button.setContentCompressionResistancePriority(.required, for: .horizontal)
+        return button
+    }()
+    
+    private let learnMoreHandler: () -> Void
+    private let importBookmarksHandler: () -> Void
```

Comment:
I've noticed that for the NTP we have utilized the delegate pattern while, for this view, we implement _callbacks_ ( to elevate the even-driven approach with the " `onSomethingHappened` ", which I like more 😇 )...I'm conscious there is no clear (and shared) pattern within the team in terms of "code style pattern" (yet perhaps 😅  ), but I wonder whether there is a specific reason for choosing this methodology here.

Reply:
Yep, makes sense to align this I think. I just picked the function closure callbacks because I find them less verbose but also I think it makes sense to align with the majority of the codebase.